.Rbuildignore

```diff
@@ -1,17 +1,10 @@
 debian
-inst/doc/html
-inst/doc/latex
-inst/doc/man
-inst/doc/jss.bst
-vignettes/jss.bst
 ^.*\.Rproj$
 ^\.Rproj\.user$
-vignettes/notyet
 doxyfile
 \.travis\.yml
 \.dir-locals\.el
 \.clang_format
-vignettes/getCurrentVersionsOfCitedPackages.R
```

Comment:
Same.  It's a helper I have, it's an .Rbuildignore entry I need keeping it does ZERO harm to your side.

Reply:
Same here, this file was moved under `vignettes/rmd` AFAICT, and also covered by the `vignettes/rmd` line below.

Reply:
Also true.  

But I would still prefer a PR to _rebuild vignettes_ (which is truly appreciated -- I should have done that last release and did not) to focus on _rebuilding vignettes_ and be minimal about it.